utils.js

```diff
@@ -0,0 +1,76 @@
+function generateSummary(core, metrics) {
+  body = '### Lava scan results\n\n';
+
+  if (Object.keys(metrics.vulnerability_count).length === 0) {
+    return body + `No vulnerabilities found!! 🎉🎉🎉\n\n`;
+  }
+
+  body += `| Severity | Findings |\n`;
+  body += `|---|---:|\n`;
+
+  if (metrics.vulnerability_count["success"] > 0) {
+    body += `| :white_check_mark: success | ${metrics.scan_count["success"]} |\n`;
+  }
+  if (metrics.vulnerability_count["critical"] > 0) {
+    body += `| :fire: critical | ${metrics.vulnerability_count["critical"]} |\n`;
```

Comment:
For the emojis I would just use circles with the standard colors. But I admit I'm pretty boring.

Reply:
done